### ISSUE TITLE
ColorBar not working in Safari

### ISSUE THREAD

**Issue by birdsarah** [bokeh/bokeh#5056]
```console
TypeError: major_coords[i].map is not a function. (In 'major_coords[i].map(function(coord) {
        return scale_length - coord;
      })', 'major_coords[i].map' is undefined)
```

<img width="1344" alt="screen shot 2016-08-30 at 10 53 27 pm" src="https://cloud.githubusercontent.com/assets/1796208/18117581/9973b5b4-6f04-11e6-8272-866edcdc315e.png">

**Comment by birdsarah** [comment]
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/map

<img width="631" alt="screen shot 2016-08-30 at 11 10 16 pm" src="https://cloud.githubusercontent.com/assets/1796208/18117998/f7964a7e-6f06-11e6-90a8-aa388aa0774d.png">


Float64Array.map is only supported in Firefox.

**Comment by canavandl** [comment]
resolved in #5059
